### PR TITLE
Release/4.2.0

### PR DESCRIPTION
**Pull request by pestOO**
Added Easy navigation in Overview page
Updated BT limitation and link
Changed coding style to [SDL Coding style](https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide)

Related issue(s): [APPLINK-26096](https://adc.luxoft.com/jira/browse/APPLINK-26096)

### CODE REVIEW

**Comment by pestOO** [comment]
@abrita, @mghiumiusliu, please review

**Comment by AnastasiyaBritanova** [comment]
@pestOO  reviewed

**Comment by mghiumiusliu** [comment]
Reviewed.

File: docs/Software Architecture Document/History/index.md [hunk 1]
```diff
@@ -31,6 +31,8 @@
 | 3.2         | 08/10/2016 | Draft           | [Sergey Levchenko]     | Add AppLaunch Component |
 | 3.3         | 08/11/2016 | Draft           | [Elisey Zamakhov]      | Use cases moved to a separate chapter <br> Add applications data storing in Process State View |
 | 3.4         | 08/16/2016 | Draft           | [Elisey Zamakhov]      | Update Diagnostics chapter with log4cxx features usage |
+| 3.5         | 09/09/2016 | Draft           | [Elisey Zamakhov]      | Change Coding style |
+| 3.6         | 21/09/2016 | Draft           | [Elisey Zamakhov]      | Update BT BlueZ limitation |
```
Comment (jacobkeeler) [inline docs/Software Architecture Document/History/index.md]: This line seems to be following a different date format from the rest of the document

**Comment by JackLivio** [comment]
@pestOO @mghiumiusliu @AnastasiyaBritanova I don't see any guides or documentation added for the new functionality implemented in 4.2 (i.e. App launching, SDL Behavior when HMI does not respond to component isReady()).

File: docs/Software Architecture Document/History/index.md (hunk 1 above)
Comment (pestOO) [inline docs/Software Architecture Document/History/index.md]: @JackLivio,
Thank you! Fixed with 8ff26d5

**Comment by pestOO** [comment]
@JackLivio,
`AppLaunch` functionality [was already delivered](https://github.com/smartdevicelink/sdl_core_guides/pull/13) in SAD - see #5 
`HMI does not respond` does not affect Architecture and related to Detail Design information, [it's also updated in guidelines](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/22)

**Comment by pestOO** [comment]
@JackLivio, 
Do you want Luxoft to merge `master` changes to `develop` branch according to GitFlow or do it your one?

**Pull request merged**